pkg/webui/components/field/group/index.js

```diff
@@ -29,13 +30,25 @@ class FieldGroup extends React.Component {
       title,
       titleComponent = 'h4',
       errors,
+      value,
+      disabled,
+      setFieldValue,
+      setFieldTouched,
```

Comment:
what about `touched` ? do we want to display errors within the `<FieldGroup />` if `touched=false`?

Reply:
I've added touch handling now. Touching a child component will result in the corresponding group value being set as touched, validation/server errors will only appear then.